packages/v4/src/content/design-guidelines/styles/typography/typography.md

```diff
@@ -22,36 +22,74 @@ We use Red Hat Display and Red Hat Text. Red Hat Display should be used for **la
+
+* **Font family, font weight, line height, and font size:** These are the styles applied to that text category in the PatternFly CSS.
+
+* **Text style:** This is the name used to refer to that text style in the PatternFly design kit.
+
+* **Font size CSS variable:** This is the CSS variable that controls the size of this text. You can customize type sizes in your application by modifying the values associated with these variables. In general, this is not advised, and should only be done with caution, as changes to text style can have far reaching side effects on spacing within certain components.
+
 <TitleLevel
   asGrid
-  title="First level title*"
-  note="*Not to be used in content block (such as landing pages and login screens)"
+  title="Super hero heading (4xl)"
+  note="Use only for extra large titles in cards or similar locations. Do not use for page titles"
+  styleProps={styleProps.superHero} />
+<TitleLevel
+    asGrid
+    title="Hero heading (3xl)"
+    note="Use only for extra large titles in cards or similar locations. Do not use for page titles"
+    styleProps={styleProps.hero} />
+<TitleLevel
+  asGrid
+  title="First level heading (2xl, h1)"
+  note="Only one first level/h1 heading should ever exist on a page"
   styleProps={styleProps.first} />
-<TitleLevel asGrid title="Second level title" styleProps={styleProps.second} />
-<TitleLevel asGrid title="Third level title" styleProps={styleProps.third} />
-<TitleLevel asGrid title="Fourth level title" styleProps = {styleProps.fourth} />
+<TitleLevel asGrid title="Second level heading (xl, h2)" styleProps={styleProps.second} />
+<TitleLevel asGrid title="Third level heading (lg, h3)" styleProps={styleProps.third} />
+<TitleLevel asGrid title="Fourth level heading (md, h4)" styleProps = {styleProps.fourth} />
 <TitleLevel
   asGrid
-  title="Body*"
-  note="*Some components use RedHatText at 700 font weight, which is RedHatText Medium (such as alerts and navigation)"
+  title="Default body text"
+  note="This is the default text style for paragraphs, lists, tables, etc. Some components use RedHatText at 700 font weight, which is RedHatText Medium (such as alerts and navigation)"
   styleProps = {styleProps.body} />
 <TitleLevel asGrid title="Small text" styleProps = {styleProps.small} />
-<TitleLevel 
-  asGrid 
-  title="Tiny text*"
-  note="*Not to be used in content blocks (only used with data visualizations when 14px is not small enough)"
+<TitleLevel
+  asGrid
+  title="Tiny text"
   styleProps = {styleProps.tiny} />
 <TitleLevel
   className="ws-typography-code-font"
-  asGrid 
-  title="Code*"
-  note="*Used for code blocks "
+  asGrid
+  title="Code"
+  note="Used for code blocks "
   styleProps = {styleProps.code} />
 
-## Line height
-<div style={{marginBottom: '32px'}}>When measuring spacing in text-based content, plan for line height.
+  ### Customizing heading levels
```

Comment:
I wonder if this whole section belongs on the typography page when it's really talking about the title component?

Reply:
That's a fair question @mmenestr . But my feeling is that if someone is looking for this information, they are more likely to find it here than to know they need to reference the Title component. What if I add something to Title design guidelines that just links back to this for more information about heading style usage?

Reply:
Yes that works!^